helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -1123,10 +1123,14 @@ service:
   annotations: {}
 
 ################################################################################
+# Notice: Network Explorer related features should be considered to have an
+# ALPHA maturity level and be experimental. There may be breaking changes or
+# Network Explorer features may be replaced entirely with a better alternative
+# in the future.
```

Comment:
This needs review and initiative by product, this is not something we can assert here.

Reply:
Had a discussion about this with Borko and Antoine yesterday. From a product standpoint - this is not advisable at the current moment since Network Explorer went GA earlier this year and customers are using the capability at the moment. The alpha label will conflict with our messaging and lead to concern/possibly churn if escalated. It is best we call out that linux kernel version 6 is not supported at the moment as highlighted here https://github.com/signalfx/splunk-otel-collector-chart/pull/976/files at the moment as we look into https://splunk.atlassian.net/browse/OTL-2343 and escalate to leadership in parallel.